src/ukify/ukify.py

```diff
@@ -587,6 +601,25 @@ def pe_add_sections(uki: UKI, output: str):
 
     pe.write(output)
 
+def merge_sbat(input: [pathlib.Path]) -> str:
+    sbat = []
+
+    for f in input:
+        try:
+            pe = pefile.PE(f, fast_load=True)
+        except pefile.PEFormatError:
+            print(f"{f} is not a valid PE file, not extracting SBAT section.")
+            continue
+
+        for section in pe.sections:
+            if section.Name.rstrip(b"\x00").decode() == ".sbat":
+                split = section.get_data().rstrip(b"\x00").decode().splitlines()
+                # Filter out the sbat line, we'll add it back later, there needs to be only one and it
+                # needs to be first.
+                sbat += [line for line in split if not line.startswith('sbat,')]
```

Comment:
hmm, shouldn't we validate the .sbat data at least superficially, i.e. that the sbat thing is indeed the first line, and only the first line? i.e. maybe have a parse_sbat() helper that splits up the data, validates the first line, removes it, and returns the rest?

Reply:
We don't validate anything else in here - kernel, initrd, osrel, cmdline, all of these need to have some format but we just take them for good. This is a build tool, it doesn't process untrusted data, so I don't think there's much value in doing that. If the kernel and/or stub have .sbat data, whatever is adding that data should do the validation instead.

Reply:
the others are components, the first line is not a component, it describes the file format used. Hence I think it deserves different handling. I mean, you are giving it special handling *already*. I am just saying we should give even tighter handling, so that if somebody embedds their fstab as sbat data we'll fail cleanly.

Reply:
I've added some very minimal validation of the first line, I don't think it's worth to do anything more complicated than this